public_html/includes/classes/scheduler.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+class scheduler {
```

Comment:
Why are none of the SQL things namespaced in the scheduler class.  Even though they may be done using the command line the classes are still available.